numpy/random/mtrand/mtrand.pyx

```diff
@@ -23,6 +23,7 @@
 
 include "Python.pxi"
 include "numpy.pxd"
+include "cpython/pycapsule.pxd"
```

Comment:
I'm curious to see if this path is portable to Windows. I suspect not and wonder why the path isn't in the normal cython search path.